### DISCUSSION TITLE
NO Fibra NO Party

### DISCUSSION THREAD

**Discussion by nromagna** [indomus/forum#432] [HUB: Home Assistant]
Titolo schersozo... ma facilmete capibile.

L'altro giorno sono rimasto senza connessione per alcune ore ed in quel periodo NON sono riuscito a collegarmi ad HA.
L'app mi dava attendere e poi controlla la connessione.

```
URL esterno    https://xxxxxxxxx.duckdns.org:8123/config/core
URL interno    https://10.10.8.121:8123
```

Idee? Oppure vi serve qualche dato in più?

Grazie.

**Comment by Scialla** [comment]
Scusa ma non capisco la domanda... se non hai connessione come pensi di poter raggiungere casa?
Dovresti avere una connessione di backup, magari con una SIM card

**Reply by nromagna** [reply]
Effettivamente mi son dimenjticato di scrivere che ero a casa... collegato in wifi. Quindi in teoria l'app avrebbe dovito prendere come indirizzo NON quello esterno ma quello interno...

**Reply by Scialla** [reply]
Non puoi andare in https su un ip interno, i certificati vengono rilasciati sul nome del dominio e non sull'ip interno...
Quindi dovresti aggiungere l'eccezione al browser che, altrimenti, ti blocca. L'app non va così...
Per risolvere dovresti usare un reverse proxy, ad esempio caddy, c'è la guida sul sito
